docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/bitbucket/index.md

```diff
@@ -0,0 +1,120 @@
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+The Bitbucket webhook receiver will need to reference a Kubernetes `Secret` with a
+`secret` key in its data map. This [shared
+secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by Bitbucket to
+sign requests. The receiver will use it to authenticate those requests by
+verifying their signatures.
+
+:::note
+The following command is suggested for generating a complex secret:
+
+```shell
+openssl rand -base64 48 | tr -d '=+/' | head -c 32
+```
+:::
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: bb-wh-secret
+    namespace: kargo-demo
+stringData:
+    secret: <your-secret-here>
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: bb-wh-receiver
+      bitbucket:
+        secretRef:
+          name: bb-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the configuration. We can obtain 
+this URL using the following command:
+
+```
+kubectl \
+    get projectconfigs \
+    kargo-demo \
+    -n kargo-demo \
+    -o=jsonpath='{.status.webhookReceivers}'
+```
+
+## Registering with Bitbucket
+
+1. Navigate to `https://bitbucket.org/<workspace>/<repository_name>/admin/webhooks` where `<workspace>` has been replaced with your Bitbucket workspace for which you are an administrator and `repository_name` has been replaced with the name of a repository belonging to that workspace.
+
+1. Click the <Hlt>Add webhook</Hlt> button to create a webhook for the repository.
+
+1. On the <Hlt>Add new webhook</Hlt> page
+   
+   1. Enter a <Hlt>Title</Hlt> with a short description.
+   
+   1. Enter the <Hlt>URL</Hlt> that is retrieved from the [Retrieving the Receiver's URL](#retrieving-the-receivers-url) section.
+
+   1. (Recommended) Set the <Hlt>Secret</Hlt> field to the value we assigned the `secret` key 
+  from the [Configuring the Receiver](#configuring-the-receiver) step.
+    
+    :::note
+    If you generate the secret directly from the Bitbucket UI, make sure to configure the
+    secret value generated by Bitbucket as the `secret` key described in the [Configuring the Receiver](#configuring-the-receiver) section.
+    :::.
+   
+   1. (Optional) If you're using a self-signed certificate and want to disable certificate verification, select <Hlt>Skip certificate verification</Hlt>.
+   
+   :::note
+   It is recommended to not disable certificate validation since disabling certificate validation makes your vulnerable to man-in-the-middle attacks regardless of whether certificates are self-signed or CA-signed.
+   :::
```

Comment:
I would just not mention this.

I'm sick of telling users this is a bad thing.

The default (unchecked) is sensible.

Let's just not bring it up.